File: practical_5/largest.c
#include<stdio.h>

int main() {
    int a, b, c;

    printf("Enter the first number: ");
    scanf("%d", &a);

    printf("Enter the second number: ");
    scanf("%d", &b);

    printf("Enter the third number: ");
    scanf("%d", &c);

    if (a > b && a > c) {
        printf("The largest number among the 3 is %d\n", a);
    }
    else if (b > c) {
        printf("The largest number among the 3 is %d\n", b);
    }
    else {
        printf("The largest number among the 3 is %d\n", c);
    }

    return 0;
}

File: practical_5/largest.py
a = int(input("Enter the first number: "))
b = int(input("Enter the second number: "))
c = int(input("Enter the third number: "))

if (a > b and a > c):
    print("The largest number among the 3 is", a)
elif (b > c):
    print("The largest number among the 3 is", b)
else:
    print("The largest number among the 3 is", c)
